src/redux/store.js

```diff
@@ -0,0 +1,20 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
```

Comment:
- [ ] Great job so far! I've noticed that your `store` configuration doesn't match the [`redux-thunk`](https://github.com/reduxjs/redux-thunk?tab=readme-ov-file#manual-setup) documentation. I recommend taking a closer look at the documentation and configuring your app accordingly. 
![image](https://github.com/AlselwiAisha/hello-react-front-end/assets/55840999/2da66f37-8a09-43c8-827b-b9c5f9afdcb3)
- [ ] Additionally, I suggest checking your project locally and addressing any new errors that may arise.